unlock-app/src/components/interface/keychain/KeyInfoDrawer.tsx

```diff
@@ -246,9 +236,9 @@ export const KeyInfo = ({
         )}
         {subscription && (
           <KeyRenewal
-            possibleRenewals={subscription.possibleRenewals}
-            approvedRenewals={subscription.approvedRenewals}
-            balance={subscription.balance}
+            possibleRenewals={subscription.possibleRenewals!}
+            approvedRenewals={subscription.approvedRenewals!}
+            balance={subscription.balance as any}
```

Comment:
ts error in previous PR?

Reply:
No. It's using generated types now since generated service.

Reply:
I will try to fix the generated types but it's mostly out of my control.